docs/topo.md

```diff
@@ -0,0 +1,421 @@
+---
+layout: default
+title: Tesselace Index
+---
+The grounds that appear in the TesseLace index are the result of research by [Veronika Irvine](https://linktr.ee/tesselace/) and were generated using a [mathematical model](https://tesselacedotcom.wordpress.com/research/jmm2014/).  Some of these grounds are familiar, traditional grounds.  Others have not been found in known catalogues and are likely new.
```

Comment:
This paragraph is copy-pasted from `docs/_includes/intro.md`
I´d like some feed back before splitting it.

Reply:
no comments

Reply:
What do you mean by "splitting the include"?